.github/workflows/fmt.yml

```diff
@@ -0,0 +1,28 @@
+name: cargo fmt
+on:
+  pull_request:
+    paths:
+      - "**/*.rs"
```

Comment:
Love this.
It inspired me to make actions that add labels that i have been adding manually. (They are needed for changelog generation)